int32 ControlMode
float32 MeasuredTiltDot
float32 MeasuredTurnRate
float32 FilteredTiltDot
float32 FilteredTurnRate
float32 Tilt
float32 TiltRate
float32 Heading
float32 TurnRate
float32 ForwardVelocity
float32 DesiredVelocity
float32 DesiredTilt
float32 DesiredTurnRate
float32 LeftMotorPosition
float32 LeftMotorVelocity
float32 RightMotorPosition
float32 RightMotorVelocity
float32 TiltEffort
float32 HeadingEffort
float32 LeftMotorEffortCmd
float32 RightMotorEffortCmd